docs/pages/blog/next-intl-3-0.mdx

```diff
@@ -119,7 +119,7 @@ Along with the release candidate also comes a [preview of the updated docs](http
 We're still looking for more feedback, please try out the release candidate and [reach out](https://github.com/amannn/next-intl/pull/149)!
 
 ```
-npm install next-intl@3.0.0-rc.9
+npm install next-intl@3.0.0-rc.10
```

Comment:
```suggestion
npm install next-intl@next
```
to not have to update it manually

Reply:
Good point, thanks! On the very top of the list for creating such a long-running branch again is definitely setting up a proper canary release channel with full automation! 😄

Reply:
What I mean is It's already configured. You just should edit this line to the `next-intl@next` instead of hard-coding the version. 
![image](https://github.com/amannn/next-intl/assets/82659015/39b4e62e-15ca-4d13-b897-8ea90bcb0c19)